nautobot/dcim/templates/dcim/device.html

```diff
@@ -6,6 +6,15 @@
 {% load plugins %}
 
 {% block content %}
+<style>
+    span.ip_hover_button {
+        display: none;
+    }
+
+    span.ip_hover:hover span.ip_hover_button {
+        display: inline;
+    }
+</style>
```

Comment:
Please indent this block by 4 spaces (matching the `<div>` below).

Alternately, is there a reason not to define these two as generic styles (maybe `span.btn_on_hover` and `span.btn_on_hover button`?) in `nautobot/project-static/css/base.css` so that we don't need to repeat custom CSS on each page where this functionality is desirable?

Reply:
👍

If you go that route @jvanderaa I suggest doing something like this (in `nautobot/project-static/css/base.css`):

```css
span.hover_copy span.hover_copy_button {
    display: none;
}
span.hover_copy:hover span.hover_copy_button {
    display: inline;
}
```

Then all copy buttons would have the `hover_copy_button` class and the wrapper (`ip_hover` in your current code base) would become `hover_copy`.